app/components/channel_list/index.test.tsx

```diff
@@ -3,16 +3,25 @@
 import Database from '@nozbe/watermelondb/Database';
 import React from 'react';
 
+import {MM_TABLES} from '@app/constants/database';
+import {TeamModel} from '@app/database/models/server';
 import {renderWithEverything} from '@test/intl-test-helper';
 import TestHelper from '@test/test_helper';
 
 import ChannelsList from './index';
 
 describe('components/channel_list', () => {
-    let database: Database | undefined;
+    let database: Database;
     beforeAll(async () => {
         const server = await TestHelper.setupServerDatabase();
         database = server.database;
+
+        await database.write(async () => {
+            const team = (await database.get<TeamModel>(MM_TABLES.SERVER.TEAM).query().fetch()).pop();
+            await team?.update(() => {
+                team.displayName = 'Test Team!';
+            });
+        });
```

Comment:
@avinashlng1080 / @enahum - not sure if this is the best way to run the test against snapshot with "static" db values; I notice team name is dynamic (will change on every test) from [here](https://github.com/mattermost/mattermost-mobile/blob/gekidou/test/test_helper.js#L183)

Reply:
this would work but:
1. Do you need a static name?
2. Can you make it so that you pass the name to the `fakeTeam` generator so it sets the value that you need/want ?